packages/commonwealth/client/styles/pages/search/search_bar.scss

```diff
@@ -16,7 +16,7 @@
       @include b1;
 
       border: 1px solid $neutral-300;
-      border-radius: $border-radius-rounded-corners;
+      border-radius: 6px;
```

Comment:
not sure if we really need this?

Reply:
@mzparacha If we don't add a border-radius of 6px then the search bar had 90 degree right angles, which isn't in the Figma designs. I used the same border-radius as the Search Common search bar.